Dragster.h

```diff
@@ -1,24 +1,36 @@
+/*
+ * This file is a part of Dragster car set library.
+ *
+ * Defines: dragster object
```

Comment:
```suggestion
 * Declares the Dragster class used to perform basic interactions
 * with the robot (https://amperka.ru/product/dragster)
 *
```

Class ≠ Object. Declaration ≠ Definition. And let the product link be there (albeit it’s only available in RU for now)